srcpkgs/musca/template

```diff
@@ -1,19 +1,20 @@
 # Template file for 'musca'
 pkgname=musca
 version=0.9.24
-revision=3
+revision=4
 build_style=gnu-makefile
 makedepends="libX11-devel libXcursor-devel"
 short_desc="Tiling window manager, with features nicked from ratpoison and dwm"
 maintainer="Diogo Leal <diogo@diogoleal.com>"
 license="GPL-3.0-or-later"
-homepage="http://aerosuidae.net/musca.html"
-distfiles="https://distfiles.voidlinux.de/musca-${version}/musca-${version}.tgz"
+homepage="https://linux.die.net/man/1/musca"
+distfiles="https://ftp.netbsd.org/pub/pkgsrc/distfiles/musca-${version}.tgz"
```

Comment:
https://launchpad.net/musca seems to still work

Reply:
Oh, I didn't notice. I wasn't able to find any release archives in launchpad, so I'm leaving `distfiles` intact (there are revisions in the repo and there is a link to download a tarball of a revision, but the request returns `500 Internal Server Error`)